AzureAutomationAuthoringToolkit/AzureAutomationAuthoringToolkitInner.psm1

```diff
@@ -170,22 +177,10 @@ function Install-AzureAutomationIseAddOn {
     else {
          Write-Verbose "AzureAutomationAuthoringToolkit: Content already present. Not adding any content."
     }
-
```

Comment:
Shouldn't we call Copy-ConfigFile in here? Also, can we rename it to Copy-AzureAutomationAuthoringToolkitConfigFile, since it will be an exposed cmdlet to users?